etc/schema.json

```diff
@@ -999,6 +999,10 @@
                     "description": "DNS opcode as an integer",
                     "type": "integer"
                 },
+                "tc": {
+                    "description": "A 1-bit subfield for truncated response that specifies if the length of the message exceeds the allowed length",
```

Comment:
I'd like others' opinions on this but I think this looks very RFC definition and not very Suricata specific like what does this field in Suricata logs represent. Like `DNS truncated response field as a boolean`..
Note that this is not a change requested. Would like your and other opinions on this.

Reply:
I do not think there is a difference for this field between RFC and Suricata...

If I want to be nitpicky, I would say `1-bit subfield` is redundant

Reply:
Is your question towards description style? If so, I understand what you say. 
Have we ever discussed what we'd like these descriptions to look like? Any style guide or something?

Reply:
I think it should be pretty simple, perhaps along the lines of:
`DNS truncation flag`.

We should actually say that this means the DNS truncation message was truncated.  Its easy to set this value to true or false whether the actual DNS message was truncated or not.  So this should just describe what that header value was set to.